d2core/d2render/ebiten/mask.go

```diff
@@ -0,0 +1,39 @@
+package ebiten
+
+import (
+	"image"
+	"image/color"
+
+	"github.com/hajimehoshi/ebiten/v2"
+)
+
+type Mask struct {
+	drawDebug  bool
+	background *ebiten.Image
+	alpha      *ebiten.Image
+}
```

Comment:
Would we need this if we implement `d2interface.DrawSubSurface`?

Reply:
same as for the interface